locals.tf

```diff
@@ -1,5 +1,7 @@
 locals {
-  accounts = { for account in csvdecode(file("${path.module}/accounts.csv")) : account.name => account }
+  accounts          = { for account in csvdecode(file("${path.module}/accounts.csv")) : account.name => account }
+  oneagent_features = { for oneagent_features in csvdecode(file("${path.module}/support_files/oneagent_features.csv")) : oneagent_features.key => oneagent_features }
 
   aws_environments = var.environment == "production" ? ["production"] : ["build", "staging", "integration"]
+  is_production    = contains(local.aws_environments, terraform.workspace)
```

Comment:
Surely this would be better as `var.environment == "production"`, I'm not sure what this check is doing otherwise. :thinking:

Reply:
I will modify this code to enhance readability.
`is_production    = var.environment == "production"`